autoload/airline/extensions/anzu.vim

```diff
@@ -7,7 +7,7 @@ function! airline#extensions#anzu#init(ext)
 endfunction
 
 function! airline#extensions#anzu#apply(...)
-	call airline#extensions#prepend_to_section("z", "%{anzu#search_status()}")
-" 	let w:airline_section_z = ' %{anzu#search_status()} ' . get(w:, "airline_section_z", "") . get(g:, "airline_section_z", "")
+	let g:anzu_status_format='[%i/%l]'
```

Comment:
that was there, to always force updating the cache so that vim-airline will always show on which item the cursor is (but do not echo the search term as this might be quite long). Now you broke seamless integration with vim-airline again and the search statistics is not displayed in the statusline anymore.